services/apps/test.sh

```diff
@@ -19,7 +19,7 @@ if [ $(docker ps -a | grep de-db | wc -l) -gt 0 ]; then
     docker rm de-db
 fi
 
-docker run --name de-db -e POSTGRES_PASSWORD=notprod -d -p 5432:5432 discoenv/de-db
+docker run --name de-db -e POSTGRES_PASSWORD=notprod -d -p 35432:5432 discoenv/de-db
```

Comment:
Changing the Postgres port allows the tests to run on a machine where Postgres is already running.